common/source/docs/common-imu-batchsampling.rst

```diff
@@ -92,6 +92,10 @@ Advanced Configuration and Analysis
 - :ref:`INS_LOG_BAT_LGIN <INS_LOG_BAT_LGIN>` interval between pushing samples to the dataflash log, in ms.  Increase this to reduce the time taken to flush data to the dataflash log, reducing cycle time.  This will be at the expense of increased system load and possibly choking up the dataflash log for other messages
 - :ref:`INS_LOG_BAT_LGCT <INS_LOG_BAT_LGCT>` Number of samples to push to count every :ref:`INS_LOG_BAT_LGIN <INS_LOG_BAT_LGIN>` ms.  Increase this to push more samples each time they are sent to the dataflash log.  Increasing this may cause timing jitter, and possibly choke up the dataflash log for other messages
 
+.. note:: On an H7 based autopilot such as the CubeOrange or Zealot H743, raw IMU logging can be used and will provide better data for analysis. Set INS_LOG_BITMASK = 0 and sent the raw IMU bit (bit 19) in LOG_BITMASK. Raw logging is very helpful when doing filter tuning, but will give you really large logs which generally means you don't want to leave it on. Batch logging on the other hand can be left on all the time which helps with ad-hoc analysis. 
```

Comment:
params should be linked

Reply:
not sure I agree with leaving batch logging on all the time

Reply:
updated